libmscore/lyrics.h

```diff
@@ -126,6 +126,8 @@ class LyricsLine final : public SLine {
 
       Lyrics* lyrics() const                          { return toLyrics(parent());   }
       Lyrics* nextLyrics() const                      { return _nextLyrics;         }
+      bool isEndMelisma() const                       { return lyrics()->ticks().isNotZero(); }
+      bool isDash() const                             { return lyrics()->ticks().isZero();    }
```

Comment:
Using `!isEndMelisma` is better to reduce number of code duplication.

Reply:
Done